.github/workflows/perf.yml

```diff
@@ -135,3 +158,14 @@ jobs:
           if [ -s heaptrack_scap_diff_above_thresh.txt ]; then
             exit 2
           fi
+
+      # Check will fail if there any google benchmark is slowed more than 5%
+      - name: Check >= 5% slowdown on google benchmarks
```

Comment:
For every benchmark in our diff, check if diff in percentage is above 5% and eventually trigger a failure.